tests/math/testMultiStatistics.cpp

```diff
@@ -0,0 +1,124 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+/**
+ * @file testMultiStatistics.cpp
+ * @ingroup Tests
+ * @author Bertrand Kerautret (\c kerautre@loria.fr )
+ * LORIA (CNRS, UMR 7503), University of Nancy, France
+ *
+ * @date 2015/11/10
+ *
+ * Functions for testing class MultiStatistics.
+ *
+ * This file is part of the DGtal library.
+ */
+
+///////////////////////////////////////////////////////////////////////////////
+#include <iostream>
+#include "DGtal/base/Common.h"
+#include "ConfigTest.h"
+#include "DGtalCatch.h"
+#include "DGtal/helpers/StdDefs.h"
+#include "DGtal/math/MultiStatistics.h"
+///////////////////////////////////////////////////////////////////////////////
+
+using namespace std;
+using namespace DGtal;
+
+///////////////////////////////////////////////////////////////////////////////
+// Functions for testing class MultiStatistics.
+///////////////////////////////////////////////////////////////////////////////
+
+TEST_CASE( "Testing MultiStatistics" )
+{  
+  MultiStatistics stats (100, false);
+  for(unsigned int i = 0; i< 100; i++)
+    {
+      for(unsigned int j = 0; j<= i; j++)
+        {
+          stats.addValue(i, j);
+        }
+    }
+  stats.terminate();
+    
+  SECTION("Testing feature math of MultiStatistics without saving data")
+    {
+      unsigned int val = 50;
+      REQUIRE( (stats.max(val) == val) );      
```

Comment:
please remove the  double (), you don't need them anymore and it will prevent nice Catch outputs of the test fails otherwise